src/github.com/getlantern/autoupdate-server/server/signature.go

```diff
@@ -0,0 +1,85 @@
+package server
+
+import (
+	"crypto"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"encoding/hex"
+	"encoding/pem"
```

Comment:
In case it's of any use to you, you can use our keyman library for doing crypto/certificate stuff - https://github.com/getlantern/keyman